### PR TITLE
Annuities calculator

### PR DESCRIPTION
**Pull request by dwhenry**
Prototype implementation for testing

### CODE REVIEW

**Comment by benbarnett** [comment]
Just a reminder to push the monthly -> annual change up when you can.

Think it just makes sense as we'll forget the context over the weekend. Well I will anyway.

**Comment by dwhenry** [comment]
@benbarnett rename done. 😄

File: app/views/calculators/annuity/_calculator.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%= render 'form', form: form %>
+<% if form.valid? %>
+  <%= render 'estimate',
+    estimate: form.estimate,
+    age: form.age.to_i,
+    pension_income: form.pension_income,
+    life_expectancy: form.life_expectancy.to_i,
+    valuation: form.estimate[:valuation]
```
Comment (benbarnett) [inline app/views/calculators/annuity/_calculator.html.erb]: We're passing the whole `estimate` to this partial – is it worth just referencing it from there?

Comment (dwhenry) [inline app/views/calculators/annuity/_calculator.html.erb]: this is following the established pattern. I can see some benefit - not having to have the long accessor where it is used multiple times.

But agree it most likely isn't worth it in this case, but would suggest we revisit it when we make this production ready.

Comment (benbarnett) [inline app/views/calculators/annuity/_calculator.html.erb]: 🆗 👌

**Comment by benbarnett** [comment]
More than enough for a prototype! 👍

**Pull request merged**